common/lib/xmodule/xmodule/js/spec/video/video_speed_control_spec.js

```diff
@@ -38,21 +38,11 @@
                         expect($(link)).toHaveData(
                             'speed', state.speeds[index]
                         );
-                        expect($(link).find('a').text()).toBe(
+                        expect($(link).find('.speed-link').text()).toBe(
```

Comment:
Good question: `speed-button` is the main control, while `speed-link` are individual options within the Speed menu. They were previously links so maintaining the "link" reference might've been overlooked.